core/modules/file/views/file.views.inc

```diff
@@ -177,6 +177,17 @@ function file_views_data() {
      ),
   );
 
+  // Icon based on file extension.
+  $data['file_managed']['icon'] = array(
+    'title' => t('Icon'),
```

Comment:
Can we please call this "File type icon"?

Reply:
That's a ... longer string that provides no better info in views admin UI, I think. :wink: